test/colony-network-mining.js

```diff
@@ -945,19 +966,16 @@ contract("ColonyNetworkMining", accounts => {
       await giveUserCLNYTokensAndStake(colonyNetwork, MAIN_ACCOUNT, DEFAULT_STAKE);
       await giveUserCLNYTokensAndStake(colonyNetwork, OTHER_ACCOUNT, DEFAULT_STAKE);
 
-      await advanceMiningCycleNoContest(colonyNetwork, this);
-
-      await giveUserCLNYTokens(colonyNetwork, MAIN_ACCOUNT, DEFAULT_STAKE);
-      await giveUserCLNYTokens(colonyNetwork, OTHER_ACCOUNT, DEFAULT_STAKE);
-      await giveUserCLNYTokens(colonyNetwork, OTHER_ACCOUNT2, DEFAULT_STAKE);
+      await fundColonyWithTokens(metaColony, clny, INITIAL_FUNDING.muln(3));
+      await setupFinalizedTask({ colonyNetwork, colony: metaColony });
+      await setupFinalizedTask({ colonyNetwork, colony: metaColony });
+      await setupFinalizedTask({ colonyNetwork, colony: metaColony });
```

Comment:
I kinda wanted to drop these unnecessary tasks being created now we can `giveUserCLNYTokens` without creating tasks. Each test should rely on the minimum amount of log entries to reproduce the tested behaviour.

Reply:
In this case though these calls are not giving tokens but rather specifically setting up the reputation log. The test runs through the dispute process explicitly so I think we need exactly this many reputation entries.

Reply:
The reputation log in almost all of the cases doesn't need these additional items in to be setup. We should aim to only create the minimum necessary reputation for a unit test.